swift/extractor/trap/TrapLabel.h

```diff
@@ -56,8 +59,10 @@ inline auto trapQuoted(const std::string& s) {
   return std::quoted(s, '"', '"');
 }
 
-template <typename Tag>
+template <typename TagParam>
 struct Binding {
+  using Tag = TagParam;
```

Comment:
What's the reason for changing this?

Reply:
this allows to get a tag relevant for an `Entry` with `Entry::Tag`, which is needed in some functions using `Entry` as a template parameter. But to be honest, I don't remember any more where I used that (or if it's still used in the current state of the code). Still, accessing template parameters with nested type aliases is a common enough requirement when dealing with templates.

Reply:
Ah, I had not immediately realized that `Tag` is public. I'd personally wait with introducing this until I really need it.

Reply:
done